cmd/kots/cli/util.go

```diff
@@ -67,5 +67,5 @@ func extractPreferredChannelSlug(upstreamURI string) (string, error) {
 	if replicatedUpstream.Channel != nil {
 		return *replicatedUpstream.Channel, nil
 	}
-	return "stable", nil
+	return "", nil
```

Comment:
this shifts the responsibility on the caller to explicitly fallback back to stable, which isn't ideal. how about just returning a boolean to indicate if the upstream URL had a channel specified?
```suggestion
	return "stable", false, nil
```

Reply:
actually, another alternative is to pass the logger as a param and just print the message in this function if the logger isn't nil.

Reply:
👍 went the logger route